packages/account-tree-controller/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **BREAKING:** Bump peer dependency `@metamask/snaps-controllers` from `^12.0.0` to `^14.0.0` ([#6035](https://github.com/MetaMask/core/pull/6035))
 - Bump `@metamask/snaps-sdk` from `^7.1.0` to `^9.0.0` ([#6035](https://github.com/MetaMask/core/pull/6035))
 - Bump `@metamask/snaps-utils` from `^9.4.0` to `^11.0.0` ([#6035](https://github.com/MetaMask/core/pull/6035))
+- Export ID conversions functions and constants ([#6062](https://github.com/MetaMask/core/pull/6062))
```

Comment:
```suggestion
- Properly export `AccountWalletCategory` constant ([#6062](https://github.com/MetaMask/core/pull/6062))
```

Actually it was wrongly exported as a type, but given it's an enum, we should export it as a "value".